skiko/buildSrc/src/main/kotlin/properties.kt

```diff
@@ -1,27 +1,44 @@
 import org.gradle.api.Project
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
 import java.io.File
 
-enum class OS(val id: String, val clangFlags: Array<String>) {
+enum class OS(
+    @get:Input
```

Comment:
Why this attribute?

Reply:
I thought it was necessary, so Gradle would include the actual values of the ids and the flags into up-to-date checks (of WasmCrossCompile tasks).
However, I thought about it again, and now I believe it is not necessary, because if an id or a flag is changed in code, then changes in script's runtime classpath invalidate all tasks.
I will remove the unnecessary annotations.